estate/models/estate_property.py

```diff
@@ -0,0 +1,32 @@
+from odoo import fields, models
+
+
+class Property(models.Model):
+    _name = "estate.property"
+    _description = "Real Estate Application"
+
+    name=fields.Char(string="Name",required=True)
+    description=fields.Text(string="Description")
+    postcode=fields.Char(string="Postcode")
+    date_availability=fields.Date(string="Availability",copy=False,default=fields.Date.add(fields.Date.today(),days=90))
+    expected_price=fields.Float(string="Expected Price",required=True)
+    selling_price=fields.Float(string="Selling Price",readonly=True,copy=False)
+    bedrooms=fields.Integer(string="Number of Bedrooms",default=2)
+    living_area=fields.Integer(string="Living Area Size")
+    facades=fields.Integer(string="Facades")
+    garage=fields.Boolean(string="Garage")
+    garden=fields.Boolean(string="Garden")
+    garden_area=fields.Integer(string="Garden Area")
+    garden_orientation=fields.Selection(string="Garden Orientation",
+        selection=[("north","North"),
+                   ("south","South"),
+                   ("east","East"),
+                   ("west","West")]
+                   )
+    active=fields.Boolean(string="Active",default=True)
+    state=fields.Selection(copy=False,required=True,default="new",string="State",
+                           selection=[("new","New"),
+                                      ("offer_recieved","Offer Received"),
+                                      ("offer_accepted","Offer Accepted"),
+                                      ("sold","Sold"),
+                                      ("canceled","Canceled")])
```

Comment:
```suggestion
    state=fields.Selection(
        copy=False,
        required=True,
        default="new",
        string="State",
        selection=[
            ("new","New"),
            ("offer_recieved","Offer Received"),
            ("offer_accepted","Offer Accepted"),
            ("sold","Sold"),
            ("canceled","Canceled")
        ],
    )
```
Indentation like this seems cleaner in my opinion. Same for above, such that the selections choices are aligned.